make eagle_default upload is broken

`make eagle_default upload` produces this error message:
```
PX4 CONFIG: posix_eagle_default
Scanning dependencies of target upload
make[4]: *** No rule to make target `../src/firmware/posix/build_mainapp_apps', needed by `src/firmware/posix/CMakeFiles/upload'.  Stop.
make[3]: *** [src/firmware/posix/CMakeFiles/upload.dir/all] Error 2
make[2]: *** [src/firmware/posix/CMakeFiles/upload.dir/rule] Error 2
make[1]: *** [upload] Error 2
make: *** [posix_eagle_default] Error 2
Add Comment Collapse
```

It works up to 5211da9f5b56dd140930051a2fcbe826d20a2755. The next one, e904184, produces lots of CMake errors. The following commit, 1692590, and all newer ones produce the above error.

--- a/src/firmware/posix/CMakeLists.txt
+++ b/src/firmware/posix/CMakeLists.txt
@@ -35,7 +35,7 @@ if ("${BOARD}" STREQUAL "eagle")
 			 BOARD ${BOARD}
 			 FILES ${CMAKE_CURRENT_BINARY_DIR}/mainapp
 				 ${CMAKE_SOURCE_DIR}/posix-configs/eagle/flight/mainapp.config
-			 DEPENDS build_mainapp_apps
+			 DEPENDS mainapp
 			 DEST /home/linaro)
 
 else()

--- a/src/firmware/qurt/CMakeLists.txt
+++ b/src/firmware/qurt/CMakeLists.txt
@@ -55,7 +55,7 @@ else()
 			 FILES ${CMAKE_CURRENT_BINARY_DIR}/libmainapp.so
 				 ${CMAKE_CURRENT_BINARY_DIR}/libpx4muorb_skel.so
 				 ${CMAKE_SOURCE_DIR}/posix-configs/eagle/flight/px4.config
-			 DEPENDS build_mainapp_dsp build_mainapp_dsp
+			 DEPENDS mainapp px4muorb_skel
 			 DEST /usr/share/data/adsp)
 endif()
 

